examples/angular/tools/ngsw_config.bzl

```diff
@@ -0,0 +1,38 @@
+"Angular service worker support (credits: https://github.com/marcus-sa)"
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+def ngsw_config(name, config, index_html, src, out = None, **kwargs):
```

Comment:
are we able to use `@angular/service-worker/ngsw-config` from `npm_package_bin` instead?

Reply:
Do you mean solely using `@angular/service-worker/ngsw-config` instead of wrapping it inside `nodejs_binary`?

Maybe something for @marcus-sa to look at?

Reply:
There was an issue with `npm_package_bin` at the moment of coding it with resolving data (pre v1), so I'll have a look at it and test it out.

Reply:
I've prefer for downstream users to not have to do this if we can avoid it.  
@alexeagle any ideas what might be up here?

Reply:
Ah yes I remember, since `genrule` is consuming the `nodejs_binary` as a tool, whatever `srcs` being provided to `genrule` will not be accessible at runtime for `npm_package_bin`, which means you'll need to provide the `srcs` to the actually binary (`nodejs_binary`)

Reply:
Something feels wrong-shaped there - npm_package_bin should replace the need for genrule. do you have a branch that demonstrates that problem?